Dockerfile

```diff
@@ -0,0 +1,9 @@
+ARG PYTHON_VER
+FROM python:${PYTHON_VER}
+ARG PYTHON_VER
```

Comment:
Is `ARG PYTHON_VER` on purpose twice?

Reply:
yes, because of [how ARG and FROM interact](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact).

> An ARG declared before a FROM is outside of a build stage, so it can’t be used in any instruction after a FROM. To use the default value of an ARG declared before the first FROM use an ARG instruction without a value inside of a build stage:

Reply:
Nice. Something learned today :-)